src/software/SfM/main_SfMInit_ImageListing.cpp

```diff
@@ -96,6 +125,8 @@ int main(int argc, char **argv)
     sfileDatabase = "",
     sOutputDir = "",
     sKmatrix;
+  std::string sPriorWeights;
+  std::pair<bool, Vec3> prior_w_info(false, Vec3::Zero());
```

Comment:
default must be true, (1.0,1.0,1.0) or set to this value if sPriorWeights is empty and -P is activated